src/ResetViewControl.ts

```diff
@@ -14,15 +14,22 @@ export type ResetViewControlOptions = {
    * It renders a box by default.
    */
   icon?: string;
+
+  /**
+   * "zoom" and "center" properties are useful when the map is mounted in a particular position 
+   * and you need this button to reset the view to another position.
+  */
+  zoomToReset?: number;
```

Comment:
What do you think of naming these `defaultZoom`  and `defaultCenter`?

Reply:
At the start of this contribution, I also thought of these names.
But once I saw the whole thing, I wonder if it wouldn't be ambiguous with the mapContainer's center/zoom definition.

```tsx
<MapContainer zoom={defaultMapZoom} center={defaultMapCenter}>
  <ResetViewControl
    defaultZoom={defaultZoom}
    defaultCenter={defaultCenter}
  />
</MapContainer>
```
Maybe the couple `targetZoom` / `targetCenter`?
But if you think `defaultZoom` / `defaultCenter` is better, i'm totally okay to change it!

Reply:
Any though about this @valentinogagliardi?